certified-connectors/Talkdesk/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3046 @@
+                  "x-ms-visibility": "internal"
+                },
+                "title": {
+                  "type": "string",
+                  "description": "The webhook's title",
+                  "title": "Title",
+                  "default": "When inbound call starts",
+                  "x-ms-visibility": "internal"
+                }
+              },
+              "required": [
+                "callbackurl",
+                "message",
+                "title"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/webhooks/triggers/{trigger_name}/{trigger_id}": {
+      "delete": {
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {}
+          },
+          "404": {
+            "description": "Not Found.",
+            "schema": {}
```

Comment:
@StefanCRT22 Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**